testing/test_pytest_html.py

```diff
@@ -1005,12 +1005,32 @@ def test_css(self, testdir, recwarn, colors):
             assert str(v["path"]) in html
             assert v["style"] in html
 
-    def test_css_invalid(self, testdir, recwarn):
+    @pytest.mark.parametrize(
+        "files",
+        [
+            "style.css",
+            ["abc.css", "xyz.css"],
+            "testdir.makefile('.css', * {color: 'white'}",
+        ],
+    )
+    def test_css_invalid(self, testdir, recwarn, files):
+        assert files is not None
         testdir.makepyfile("def test_pass(): pass")
-        result = testdir.runpytest("--html", "report.html", "--css", "style.css")
+        path = files
+        if isinstance(files, list):
+            file1 = files[0]
+            file2 = files[1]
+            result = testdir.runpytest(
+                "--html", "report.html", "--css", file1, "--css", file2
+            )
+        else:
+            result = testdir.runpytest("--html", "report.html", "--css", path)
         assert result.ret
         assert len(recwarn) == 0
-        assert "No such file or directory: 'style.css'" in result.stderr.str()
+        if isinstance(files, list):
+            assert files[0], files[1] in result.stderr.str()
```

Comment:
Sorry I missed this, but this doesn't do what you expect. You aren't actually checking for `files[0]`. This just checks whether or not `files[0]` is truthy and executes `files[1] in result.stderr.str()` if it's not. For example:
```
>>> l = [1,2,3]
>>> assert 0, 2 in l
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError: True
>>> assert 44, 2 in l
```